Update Azure/RealtimeBoard Tutorial

CatherineRTB opened the pull request:
Enabling SSO on Realtimeboard side has changed, updating the last step of this instruction

azurecla commented:
Hi __@billbliss__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (billbl). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@billbliss : Thanks for your contribution to the Azure documentation! The author, @dstrockis, has been notified to review your proposed change.

KatieCumming commented:
@billbliss - Hi Bill, thanks so much for catching this! Merging.

CatherineRTB committed:
Update Azure/RealtimeBoard Tutorial 

Enabling SSO on Realtimeboard side has changed, updating the last step of this instruction

ktoliver committed:
edit for clarity
